pkg/fleet-manager/backup_restore_migrate_shared_test.go

```diff
@@ -0,0 +1,431 @@
+/*
+Copyright Kurator Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+	http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fleet
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	velerov1 "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/yaml"
+
+	backupapi "kurator.dev/kurator/pkg/apis/backups/v1alpha1"
+)
+
+const backupTestDataPath = "backup-testdata/backup/"
```

Comment:
nit: ./testdata/backup/

Reply:
ok. actually, the path may changes when we add a backup dir and put all related file in it

Reply:
fix in #407